bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/checkbox.js

```diff
@@ -82,7 +82,16 @@ pimcore.object.classes.data.checkbox = Class.create(pimcore.object.classes.data.
         });
 
         return [
-            defaultField, {
+            defaultField,
+            {
```

Comment:
new field should also be considered when cloning the checkbox (same for other data types). see method "applySpecialData"

Reply:
added support for clone